tests/unit/test_tools.py

```
@@ -1423,9 +1418,11 @@ def test_transcribe_audio_correct_model(self, mock_getenv, mock_file, mock_post)
     def test_transcribe_audio_import_error(self):
         """Test audio transcription with ImportError."""
         import sys
+
         with patch.dict('sys.modules', {'requests': None}):
             result = transcribe_audio('/path/to/audio.mp3')
```

Ruff (0.14.10)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)